### ISSUE TITLE
Random Forest test not working

### ISSUE THREAD

**Issue by margaeor** [1ozturkbe/OCTHaGOn.jl#112]
The test that is used to test the RF as a constraint learner doesn't work because:
1) Right now it does not use random forest, it uses base_classifier() which is OCT-H and not RF.
2) Even if I switch from base_classifier() to base_rf_classifier() in the test, then the test doesn't run
because it seems that rf_classifier is not supported

**Comment by 1ozturkbe** [comment]
Yes, we don't have RF classifiers implemented right now (even though the struct ```base_rf_classifier``` exists), but it is very possible to implement it. Given that this is of direct interest to you research-wise, I think it might be a good opportunity for you to get to learn and contribute to the internals of the code. Let me know if this interests you.

**Comment by margaeor** [comment]
Yeah, I can implement that in the future. Right now I am working on gradient boosting
